core/io/multiplayer_api.cpp

```diff
@@ -854,7 +905,7 @@ void MultiplayerAPI::_send_rpc(Node *p_from, int p_to, bool p_unreliable, bool p
 	ERR_FAIL_COND(name_id_compression > 1);
 
 	// We can now set the meta
-	packet_cache.write[0] = command_type + (node_id_compression << 3) + (name_id_compression << 5);
+	packet_cache.write[0] = command_type + (node_id_compression << 3) + (name_id_compression << 5) + ((byte_only_or_no_args ? 1 : 0) << 6);
```

Comment:
We should probably make an enum with these metadata.

Reply:
Do you mean `<< 3`, `<< 5` ?
Yes, I agree.

Reply:
I've replaced it with a macro.